tools/python_api/test/test_scan_pandas.py

```diff
@@ -378,6 +378,7 @@ def test_scan_long_utf8_string(tmp_path: Path) -> None:
     result = conn.execute("LOAD FROM df WHERE name = '非常长的中文' RETURN count(*);")
     assert result.get_next() == [1]
 
+
```

Comment:
remove new line

Reply:
i think the convension in python is two newlines between two functions.